components/cluster-as-a-service/staging/cluster-provisioner-read-pod-logs.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Patching the resource like this will replace it entirely. Use `kustomize build components/cluster-as-a-service/staging` to verify the resulting ClusterRole contains the rules from the base rbac plus your changes.

Take a look at this patch for an example of how to append to an array: https://github.com/redhat-appstudio/infra-deployments/blob/b22e629801630a927a1b0f5aed8d6dfea8319c9c/components/cluster-as-a-service/staging/add-hypershift-params.yaml

Reply:
Alright based on the newest changes I'm adding new resources instead of editing existing ones so I shouldn't be replacing anything.